mne/preprocessing/tests/test_maxwell.py

```diff
@@ -0,0 +1,114 @@
+# Author: Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import warnings
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose,
+                           assert_array_almost_equal)
+
+from mne.preprocessing import maxwell
+from mne.datasets import testing
+from mne.io import Raw
+warnings.simplefilter('always')  # Always throw warnings
+
+
+def test_spherical_harmonic():
+    from scipy.special import sph_harm as scipy_sph_harm
+    """Test for spherical harmonics"""
+    deg = 1
+    order = -1
+    azimuth = np.random.random_sample(size=(50, 1)) * 2 * np.pi
+    polar = np.random.random_sample(size=(50, 1)) * np.pi
+
+    # Internal calculation
+    sph_harmonic = maxwell._sph_harmonic(deg, order, azimuth, polar)
+    # Check against scipy
+    sph_harmonic_scipy = scipy_sph_harm(order, deg, azimuth, polar)
+
+    assert_array_almost_equal(sph_harmonic, sph_harmonic_scipy, decimal=15,
+                              err_msg='Spherical harmonic mismatch')
+
+
+@testing.requires_testing_data
+def test_maxwell_filter():
+    """Test multipolar moment and Maxwell filter"""
+
+    # Load testing data (raw, SSS std origin, SSS non-standard origin)
+    data_path = op.join(testing.data_path(download=False))
+
+    file_name = 'test_move_anon'
+
+    raw_fname = op.join(data_path, 'SSS', file_name + '_raw.fif')
+    sss_std_fname = op.join(data_path, 'SSS', file_name +
+                            '_raw_simp_stdOrigin_sss.fif')
+    sss_nonStd_fname = op.join(data_path, 'SSS/', file_name +
+                               '_raw_simp_nonStdOrigin_sss.fif')
+
+    raw = Raw(raw_fname, preload=False, proj=False,
+              allow_maxshield=True).crop(0., 1., False)
+    raw.preload_data()
+    sss_std = Raw(sss_std_fname, preload=True, proj=False,
+                  allow_maxshield=True)
+    sss_nonStd = Raw(sss_nonStd_fname, preload=True, proj=False,
+                     allow_maxshield=True)
```

Comment:
add a test that loading with `proj=True` and running MaxFilter gives an appropriate error (using `assert_raises`)